tests/integration/repo/test_apt_sources_manager.py

```diff
@@ -38,14 +38,51 @@
 
 
 @pytest.mark.parametrize(
-    ["source_url", "source_arch"],
+    ["source_url", "repo_url", "repo_arch", "expected_url"],
```

Comment:
Could you add some `id`s to these test cases? It helps a lot with recognizing failing tests.

Reply:
good idea, done!